app/bundles/LeadBundle/Form/Type/EntityFieldsBuildFormTrait.php

```diff
@@ -204,7 +204,11 @@ function (FormEvent $event) use ($alias, $type): void {
                     ];
 
                     $emptyValue = '';
-                    if (in_array($type, [SelectType::class, MultiselectType::class]) && !empty($properties['list'])) {
+
+                    if (array_key_exists('use_nullable_yes_no_type', $options) && true === $options['use_nullable_yes_no_type'] && BooleanType::class === $type) {
+                        $type       = NullableYesNoButtonGroupType::class;
+                        $emptyValue = 'No change';
```

Comment:
"No change" must be translateable.

We can use for example `mautic.lead.form.no_change`:

```suggestion
                        $emptyValue = 'mautic.lead.form.no_change';
```

and add this to the file: `app/bundles/LeadBundle/Translations/en_US/messages.ini`:

```
mautic.lead.form.no_change="No change"
```

Reply:
Resolved with https://github.com/mautic/mautic/pull/14888/commits/e1a953a0f75c64e325c6e0b3e9487fdb8745ccfa so we can get this regression merged with the next release.